tests/PHPUnit/Unit/ConfigTest.php

```diff
@@ -390,5 +390,44 @@ public function test_forceSave_writesNothingIfThereAreNoChanges()
 
         @unlink($configFile);
     }
+    
+    public function testFromGlobalConfig()
+    {
+        $userFile = PIWIK_INCLUDE_PATH . '/tests/resources/Config/config.ini.php';
+        $globalFile = PIWIK_INCLUDE_PATH . '/tests/resources/Config/global.ini.php';
+        $commonFile = PIWIK_INCLUDE_PATH . '/tests/resources/Config/common.config.ini.php';
+        
+        $config = new Config($globalFile, $userFile, $commonFile);
+        $config->reload();
+        
+        $configCategory = $config->getFromGlobalConfig('Category');
+        $this->assertEquals('value1', $configCategory['key1']);
+    }
+    
+    public function testFromCommonConfig()
+    {
+        $userFile = PIWIK_INCLUDE_PATH . '/tests/resources/Config/config.ini.php';
+        $globalFile = PIWIK_INCLUDE_PATH . '/tests/resources/Config/global.ini.php';
+        $commonFile = PIWIK_INCLUDE_PATH . '/tests/resources/Config/common.config.ini.php';
+    
+        $config = new Config($globalFile, $userFile, $commonFile);
+        $config->reload();
+    
+        $configCategory = $config->getFromCommonConfig('Category');
+        $this->assertEquals('valueCommon', $configCategory['key2']);
```

Comment:
Dollar signs are supposed to be HTML encoded in config files, this fixes a bug/"feature" in `parse_ini_file()`.

Can you make it test against an array? ie `$this->assertEquals(array('key2' => ..., 'key3' => ...), $configCategory);`? This will test that the section only has the expected keys, in addition to testing the values of those keys.